includes/class-wc-countries.php

```diff
@@ -616,6 +616,12 @@ private function trim_formatted_address_line( $line ) {
 	 * @return array
 	 */
 	public function get_default_address_fields() {
+
+		$address_2_placeholder = __( 'Apartment, suite, unit etc.', 'woocommerce' );
+		if ( 'optional' === get_option( 'woocommerce_checkout_address_2_field', 'optional' ) ) {
+			$address_2_placeholder .= __( ' (optional)', 'woocommerce' );
```

Comment:
Move this space outside of the localised string - it will make translations easier.

Reply:
@mikejolley Can it be something like `$address_2_placeholder .= ' ' . __( ' (optional)', 'woocommerce' );` ?